web-server/v0.4/utils/datastore.js

```diff
@@ -0,0 +1,4 @@
+export default { 
+    elasticsearch: "http://es-perf44.perf.lab.eng.bos.redhat.com:9280/dsa.pbench.run.*/_search",
```

Comment:
Can you use "elasticsearch.perf.lab.eng.bos.redhat.com" port 9280 for production.  But since this public, can you use something like, "elasticsearch.example.com"?

Reply:
Just hostname for elasticsearch?

Reply:
It would be great to pull out the index prefix to a separate variable, can we do that?

Something like:
```
   prefix: "dsa.pbench"
```

Reply:
Consider add these index patterns to the datastore as well.

```
run_indices: "run.YYYY-MM",
result_indices: "result-data.YYYY-MM-DD",
tool_indices: "tool-data.%s.YYYY-MM-DD"
```